pydruid/client.py

```diff
@@ -17,13 +17,23 @@
 from __future__ import absolute_import
 
 import json
-import sys
+import re
 
 from six.moves import urllib
 
 from pydruid.query import QueryBuilder
 from base64 import b64encode
 
+try:
+    # available only in Python >= 3.5
+    from json.decoder import JSONDecodeError
```

Comment:
Looks like `JSONDecodeError` is derived from `ValueError`, so catching `ValueError` should just work and remove the need for this extra complexity.